Cargo.toml

```diff
@@ -85,3 +85,8 @@ rayon = "1.5.1"
 members = [
     "run-wasm",
 ]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+targets = ["x86_64-unknown-linux-gnu", "wasm32-unknown-unknown"]
```

Comment:
The other targets don't have any target-specific APIs, should I still add them to the list?

Reply:
I would say yes; I think they've expected at this point.

Reply:
Only added 64-bit targets, say the word if I should add anything else.